RiotSharp/RiotApi.cs

```diff
@@ -105,196 +108,80 @@ public RiotApi(IRateLimitedRequester rateLimitedRequester)
             requester = rateLimitedRequester ?? throw new ArgumentNullException(nameof(rateLimitedRequester));
```

Comment:
Could you remove c# 6 features? Those slipped through in the last few PRs and break compat.